src/kayenta/stages/kayentaStage/kayentaStage.html

```diff
@@ -122,6 +122,18 @@
       <span class="form-control-static"> seconds</span>
     </stage-config-field>
 
+    <stage-config-field label="Baseline Offset"
+                        help-key="pipeline.config.canary.baselineAnalysisOffset"
+                        field-columns="3">
+      <input
+        class="form-control input-sm"
+        ng-model="kayentaCanaryStageCtrl.stage.canaryConfig.baselineAnalysisOffsetInMins"
+        type="text"
```

Comment:
Is this a `text` input instead of `number` so that a SpEL expression might be used?

Reply:
Ah never mind, the `help-key` defined in your Deck PR answers my question!